src/abstractdataframe/show.jl

```diff
@@ -6,28 +6,28 @@ let
     global ourstrwidth
 
     """
-        DataFrames.ourstrwidth(x::Any)
+        DataFrames.ourstrwidth(x::Any, trueio::IO)
 
     Determine the number of characters that would be used to print a value.
     """
-    function ourstrwidth(x::Any) # -> Int
+    function ourstrwidth(x::Any, trueio::IO) # -> Int
```

Comment:
Better put `trueio` first as with other functions. Maybe also rename `io` to something like `buffer` so that `trueio` can be just `io`?

Reply:
OK